rbenv/Dockerfile

```diff
@@ -1,7 +1,43 @@
-FROM swiftdocker/swift
+FROM debian:jessie
+MAINTAINER David Ohayon <dohayon@twitter.com>
 
-ADD install-rbenv ./install-rbenv
+SHELL ["/bin/bash", "-c"]
 
-RUN apt-get update && apt-get install -y libssl-dev zlib1g-dev
+ENV FASTLANE_VERSION "1.108.0"
```

Comment:
why dont' you use ARG instead of ENV in this case to get flexible container building and tagging containers then?